.github/workflows/prbot.yml

```
@@ -47,7 +47,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        spec: ["cypress/integration/*", "cypress/integration/result-list/**/*", "cypress/integration/facets/**/*", "cypress/integration/search-box/**/*"]
+        spec:
+          [
+            'cypress/integration/*',
+            'cypress/integration/result-list/**/*',
+            'cypress/integration/facets/**/*',
+            'cypress/integration/search-box/**/*',
```

I don't think you need brackets here since this is YAML

```suggestion
          - 'cypress/integration/*'
          - 'cypress/integration/result-list/**/*'
          - 'cypress/integration/facets/**/*'
          - 'cypress/integration/search-box/**/*'
```